lib/transaction.js

```diff
@@ -11,6 +11,27 @@ var Transaction = require('loopback-connector').Transaction;
 
 module.exports = mixinTransaction;
 
+var mapIsolationLevel = function(isolationLevelString) {
+  var ret = 2;
+  switch (isolationLevelString) {
+    case Transaction.READ_UNCOMMITTED:
+      ret = 1;
+      break;
+    case Transaction.SERIALIZABLE:
+      ret = 4;
+      break;
+    case Transaction.REPEATABLE_READ:
+      ret = 8;
+      break;
+    case Transaction.READ_COMMITTED:
```

Comment:
I'm not sure of your logic here; what is supposed to be the value of `ret` for `Transaction.READ_COMMITTED`?